ONSdigital/dp-frontend-router
# Feature/adding filter flex service

### What

- Added route and config for new `dp-frontend-filter-flex-dataset` service
- Updated unit tests

### How to review

- Sense check
- Run this branch of `dp-frontend-router` with `make debug FILTER_FLEX_ROUTES_ENABLED=true`
- Get the latest dev branch of [dp-frontend-filter-flex-dataset](https://github.com/ONSdigital/dp-frontend-filter-flex-dataset) and in `routes.go`replace line 25 with:
```go
r.StrictSlash(true).Path("/flex/helloworld").Methods("GET").HandlerFunc(handlers.HelloWorld(*cfg))
```
- Run the filter-flex service `make debug`
- In a browser, go to (http://localhost:20000/flex/helloworld)
- For a complete sense check, make `FILTER_FLEX_ROUTES_ENABLED=false` in the router and go to the page again, you should get a 502 error

### Who can review

Anyone but me
